datakyt/db/README.md

```diff
@@ -0,0 +1,25 @@
+# PostgreSQL docker image with a database for a Datakyt project
+
+This image contains an initial database schema.
+
+## Build image
+
+To build this image simply run `docker build -t datakyt/docker-postgres:<version> .`
```

Comment:
What's `version`? Please elaborate.

Reply:
I mean the PostgreSQL version.  In our case we are using version 12.

Reply:
@MykytaKyt From docker [docs](https://docs.docker.com/engine/reference/commandline/build/):
 > --tag , -t  Name and optionally a tag in the ‘name:tag’ format

So this shouldn't be a PostgreSQL version, this tag is used to specify different docker image versions that may correspond to the git tags, for example. Say you have DataKyt 1.0 and 2.0 versions, that you can create docker images with different tags.